curvlinops/kfac.py

```diff
@@ -1070,3 +1070,156 @@ def frobenius_norm(self) -> Tensor:
                     )
         self._frobenius_norm.sqrt_()
         return self._frobenius_norm
+
+    def state_dict(self) -> Dict[str, Any]:
+        """Return the state of the KFAC linear operator.
+
+        Returns:
+            State dictionary.
+        """
```

Comment:
Should we add a warning here that for correct restoring one needs to pass the same `data` argument? (maybe also for other arguments, I saw you mentioned model parameters.)

Reply:
Update: Maybe not necessary because they are required in `from_state_dict`, which makes it obvious that if one passes different data/params, the linear operator will of course be different.